esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -175,7 +181,10 @@ def commfinal(self, request, tl, one, two, module, extra, prog):
                                                       sendto_fn_name  = sendto_fn_name,
                                                       sender     = fromemail,
                                                       creator    = request.user,
-                                                      msgtext    = body,
+                                                      msgtext = unicode(loader.get_template('email/default_email_html.txt').render(DjangoContext(
+                                                                                                                                       {'msgbdy': body,
+                                                                                                                                        'user': request.user,
```

Comment:
Sorry, I don't quite understand what the difference is / what needs to be done in order to make sure the right thing happens instead of the variables being rendered in the template?